src/Mod/Arch/ArchStructure.py

```diff
@@ -1395,3 +1450,18 @@ def getIcon(self):
 
 if FreeCAD.GuiUp:
     FreeCADGui.addCommand('Arch_Structure',_CommandStructure())
+    FreeCADGui.addCommand('Arch_StructuralSystem',_CommandStructuralSystem())
+    FreeCADGui.addCommand('Arch_StructuresFromSelection',_CommandStructuresFromSelection())
+
+    class _ArchStructureGroupCommand:
+
+        def GetCommands(self):
+            return tuple(['Arch_Structure', 'Arch_StructuralSystem', 'Arch_StructuresFromSelection'])
```

Comment:
For this we don't need to put the list inside a tuple, we can just build the tuple directly.
```py
return ('a', 'b', 'c')
```